Code/GraphMol/Depictor/Templates.h

```diff
@@ -0,0 +1,71 @@
+
+//
+//  Copyright ...
+
+#include <RDGeneral/export.h>
+#include <GraphMol/ROMol.h>
+#include <GraphMol/SmilesParse/SmilesParse.h>
+#include <GraphMol/MolOps.h>
+
+#include <iostream>
+#include <fstream>
+#include <unordered_map>
+
+class RDKIT_DEPICTOR_EXPORT CoordinateTemplates {
+ public:
+  //! returns a reference to the singleton CoordinateTemplates
+  /*
+      \return a reference to the singleton CoordinateTemplates
+
+      <b>Notes:</b>
+        - if the singleton CoordinateTemplates has already been instantiated
+          the singleton will be returned, otherwise the singleton will
+          be constructed.
+
+   */
+  static CoordinateTemplates& getRingSystemTemplates() {
+    static CoordinateTemplates template_mols;
+    return template_mols;
+  }
+
+  bool hasTemplateOfSize(unsigned int atom_count) {
+    if (m_templates.find(atom_count) != m_templates.end()) {
+        return true;
+    }
+    return false;
+  }
+
+  const std::vector<RDKit::ROMol*>& getMatchingTemplates(unsigned int atom_count) {
+    return m_templates[atom_count];
+  }
+
+ private:
+  CoordinateTemplates() {
+    // load templates into m_templates map by atom count
+    std::ifstream cxsmiles_templates ("templates.smi");
+    std::string smiles;
+    while(std::getline(cxsmiles_templates, smiles)) {
+        auto mol = RDKit::SmilesToMol(smiles);
+        RDKit::MolOps::symmetrizeSSSR(*mol);
```

Comment:
were you maybe intending to turn off sanitization? otherwise I thought that sssr happens by default in the reader.